wiki/osu!api/ru.md

```diff
@@ -0,0 +1,10 @@
+У osu! есть публичный API, который сторонние сервисы могут использовать для расширения и поддержки osu! платформы. Раньше он был более приватным, использовался только для каждого запроса, но теперь любой может использовать API, если у него есть учетная запись osu!.
+
+Вы можете найти документацию по API в [osu!api Вики] (https://github.com/ppy/osu-api/wiki). Текущая версия API использует закрытые ключи для аутентификации запросов. 
```

Comment:
the spaces between `]` and `(` need to be removed for the link to render (same goes for every other link here)

Reply:
yes, thanks